### ISSUE TITLE
Webhooks can't be created

### ISSUE THREAD

**Issue by tilsammans** [maartenvanvliet/moneybird#6]
I cannot create a new webhook via this gem. First, `Moneybird::Traits::Save` isn't included for Webhook. After I added that, the request is sent url-encoded, leading to a 400 error.

``` ruby
administration.webhooks.create url: "https://example.nl/moneybird/events"

@faraday_adapter=:net_http, @http=#<Faraday::Connection:0x0000000003cc0c38 @parallel_manager=nil, @headers={"User-Agent"=>"Faraday v0.12.2"}, @params={}, @options=#<Faraday::RequestOptions (empty)>, @ssl=#<Faraday::SSLOptions verify=true>, @default_parallel_manager=nil, @builder=#<Faraday::RackBuilder:0x0000000003cbe730 @handlers=[Faraday::Request::UrlEncoded, Faraday::Adapter::NetHttp], @app=#<Faraday::Request::UrlEncoded:0x0000000003adbc88 @app=#<Faraday::Adapter::NetHttp:0x0000000003ad90a0
```

**Comment by maartenvanvliet** [comment]
Thanks, I will look into it

**Comment by tilsammans** [comment]
I am more than happy to help maintain it also!

**Comment by maartenvanvliet** [comment]
Fixed in 7d5c372f7cf4517f91647b55006315fe853ee7b3
